packages/server/graphql/mutations/helpers/notifications/makeSlackBlocks.ts

```diff
@@ -1,20 +1,40 @@
 import generateUID from '../../../../generateUID'
 
-export const makeSection = (text: string) => ({
-  type: 'section',
-  text: {
-    type: 'mrkdwn',
-    text
+// Slack has the limit of 3000 chars per block
+const SLACK_MAX_BLOCK_CHAR_COUNT = 3000
+const truncationString = `[...]`
+
+const maybeTruncate = (text: string) => {
+  if (text.length > SLACK_MAX_BLOCK_CHAR_COUNT) {
+    return text.slice(0, SLACK_MAX_BLOCK_CHAR_COUNT - truncationString.length) + truncationString
   }
-})
+  return text
+}
 
-export const makeSections = (fields: string[]) => ({
-  type: 'section',
-  fields: fields.map((field) => ({
-    type: 'mrkdwn',
-    text: field
-  }))
-})
+export const makeSection = (text: string) => {
+  text = maybeTruncate(text)
```

Comment:
+1 rather than reassign the input, I think it would be easier to read if it's a new variable, like `const maybeTruncatedText = ...`

Reply:
@nickoferrall my thought is that `...` could be confused with real `...`